pom.xml

```diff
@@ -12,66 +12,116 @@
+    <appassembler.plugin.version>2.1.0</appassembler.plugin.version>
+    <build-helper.plugin.version>3.2.0</build-helper.plugin.version>
+    <buildnumber.plugin.version>1.4</buildnumber.plugin.version>
+    <docbkx.plugin.version>2.0.17</docbkx.plugin.version>
+    <exec.plugin.version>3.0.0</exec.plugin.version>
+    <felix-bundle.plugin.version>3.5.1</felix-bundle.plugin.version>
+    <h2spec.plugin.version>1.0.7</h2spec.plugin.version>
+    <jacoco.plugin.version>0.8.7</jacoco.plugin.version>
+    <jetty-version-txt.plugin.version>2.7</jetty-version-txt.plugin.version>
+    <license.plugin.version>4.1</license.plugin.version>
+    <maven.antrun.plugin.version>3.0.0</maven.antrun.plugin.version>
+    <maven.assembly.plugin.version>3.3.0</maven.assembly.plugin.version>
+    <maven.invoker.plugin.version>3.2.2</maven.invoker.plugin.version>
+    <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
+    <maven.checkstyle.plugin.version>3.1.0</maven.checkstyle.plugin.version>
+    <maven.clean.plugin.version>3.1.0</maven.clean.plugin.version>
     <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
     <maven.dependency.plugin.version>3.2.0</maven.dependency.plugin.version>
+    <maven.eclipse.plugin.version>2.10</maven.eclipse.plugin.version>
+    <maven.enforcer.plugin.version>3.0.0</maven.enforcer.plugin.version>
+    <maven.jar.plugin.version>3.2.0</maven.jar.plugin.version>
+    <maven.javadoc.plugin.version>3.3.1</maven.javadoc.plugin.version>
+    <maven.jxr.plugin.version>3.3.1</maven.jxr.plugin.version>
+    <maven.plugin.plugin.version>3.6.1</maven.plugin.plugin.version>
+    <maven.project-info-reports.plugin.version>3.1.2</maven.project-info-reports.plugin.version>
+    <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
+    <maven.remote-resources.plugin.version>1.7.0</maven.remote-resources.plugin.version>
     <maven.resources.plugin.version>3.2.0</maven.resources.plugin.version>
+    <maven.shade.plugin.version>3.2.4</maven.shade.plugin.version>
+    <maven.site.plugin.version>3.9.1</maven.site.plugin.version>
     <maven.source.plugin.version>3.2.1</maven.source.plugin.version>
     <maven.war.plugin.version>3.3.2</maven.war.plugin.version>
-    <maven.plugin-tools.version>3.6.1</maven.plugin-tools.version>
     <maven.install.plugin.version>3.0.0-M1</maven.install.plugin.version>
     <maven.deploy.plugin.version>3.0.0-M1</maven.deploy.plugin.version>
+    <servicemix-depends.plugin.version>1.4.0</servicemix-depends.plugin.version>
+    <versions.plugin.version>2.8.1</versions.plugin.version>
 
     <!-- testing -->
+    <awaitility.version>4.1.0</awaitility.version>
     <it.debug>false</it.debug>
     <jetty.test.version>5.8</jetty.test.version>
-    <spring-boot.version>2.1.1.RELEASE</spring-boot.version>
+    <jetty.testtracker.log>false</jetty.testtracker.log>
     <jsr250-api.version>1.0</jsr250-api.version>
-    <javax.annotation-api>1.3.2</javax.annotation-api>
```

Comment:
this need to updated in some maven plugin its as well

Reply:
It's there in line 44 of the main pom.xml

https://github.com/eclipse/jetty.project/blob/505874110edcd040aebd3c52c9de9f0ec03112b0/pom.xml#L44